app/models/activity.rb

```diff
@@ -0,0 +1,3 @@
+class Activity < ActiveRecord::Base
+  belongs_to :trail
+end
```

Comment:
The unique id of the trail, and the unique id of the activity do not correlate to one another. They are only used so the find and create method doesn't duplicate records. The two tables are associated by the table id assigned upon creation of the table and the activity table uses that as a resource for assignment setting up the belongs to.